src/main/kotlin/example/Header.kt

```diff
@@ -4,22 +4,19 @@ import csstype.*
 import example.Sizes.Header
 import kotlinext.js.jso
 import kotlinx.browser.window
-import mui.icons.material.GitHub
+import mui.icons.material.*
 import mui.icons.material.Menu
-import mui.icons.material.MenuBook
 import mui.material.*
 import mui.material.Size
-import react.FC
-import react.Props
-import react.ReactNode
+import react.*
 import react.dom.aria.AriaHasPopup.`false`
 import react.dom.aria.ariaHasPopup
 import react.dom.aria.ariaLabel
 import react.dom.html.ReactHTML
 import react.router.useLocation
-import react.useContext
 
 val Header = FC<Props> {
+    val (paletteMode, setPaletteMode) = useContext(ColorModeContext)
```

Comment:
And here you'll get a theme from `ThemeContext`
`var theme = useContext(ThemeContext)`